pkg/controller-gen/main.go

```diff
@@ -8,6 +8,7 @@ import (
 	"sort"
 	"strings"
 
+	"golang.org/x/exp/maps"
```

Comment:
The `"maps"` package already exists iirc, can we use that?

Reply:
We've discussed that on slack. I use the method Keys from maps package. And at time of this go version it wasn't available at the standard "maps" package.